### PR TITLE
Use both new Google API Keys

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
https://trello.com/c/wmtAcdpw

### Changes proposed in this pull request
Add both Google API keys

### Guidance to review
n/a

### CODE REVIEW

**Comment by timabell** [comment]
[Ensure search and compare UI is using both Google API keys](https://trello.com/c/wmtAcdpw/476-ensure-search-and-compare-ui-is-using-both-google-api-keys)

File: src/Startup.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public void ConfigureServices(IServiceCollection services)
             _logger.LogInformation("Using API base URL: " + apiUri);
             services.AddSingleton<ISearchAndCompareApi>(provider => new SearchAndCompareApi(new HttpClient(), apiUri));
             services.AddScoped<IFeatureFlags, FeatureFlags>();
-            services.AddSingleton<IGeocoder>(provider => new Geocoder(Configuration.GetSection("ApiKeys").GetValue<string>("GoogleMaps"), new HttpClient()));
+            services.AddSingleton<IGeocoder>(provider => new Geocoder(Configuration["google_cloud_platform_key_geocoding"], new HttpClient()));
             services.AddScoped<IMapProvider>(provider => new MapProvider(new HttpClientProvider(), Configuration.GetSection("ApiKeys").GetValue<string>("GoogleMapsStatic")));
```
[the remark this answers is not in the document]
Comment (dankmitchell) [reply src/Startup.cs]: I _think_ so 👀

[the remark this answers is not in the document]
Comment (defong) [reply src/Startup.cs]: @timabell `GoogleMapsStatic` is that still in use?

[the remark this answers is not in the document]
Comment (dankmitchell) [reply src/Startup.cs]: We're going to remove in another PR @defong

**Comment by timabell** [comment]
[Rotate & split google maps api keys](https://trello.com/c/0VbyhJOe/468-rotate-split-google-maps-api-keys)

**Pull request merged**